doc/Articles/Guides/Sync/SyncVars.md

```diff
@@ -121,6 +121,54 @@ public class Player : NetworkBehaviour
 }
 ```
 
+### Example Client & Server
+Below is a simple example of assigning a random color to each player when they're spawned on the server. All clients will see all players in the correct colors, even if they join later.
+Server will also fire the event.
+
+```cs
+using UnityEngine;
+using Mirage;
+
+public class Player : NetworkBehaviour
+{
+    [SyncVar(hook = nameof(UpdateColor), fireOnServer = true)]
+    Color playerColor = Color.black;
+
+    Renderer renderer;
+
+    // Unity makes a clone of the Material every time renderer.material is used.
+    // Cache it here and Destroy it in OnDestroy to prevent a memory leak.
+    Material cachedMaterial;
```

Comment:
I think `cachedMaterial` is beyond the scope of an example here about syncvars, would be simpler if it just set the colors.

Reply:
I just copied the original one and made change. Can make it simpler if need be.